app/models/episode_entry_handler.rb

```diff
@@ -90,15 +90,12 @@ def update_contents
       return
     end
 
-    final_contents = []
     new_contents.each do |c|
-      existing_content = find_existing_content(c)
+      existing_content = episode.find_existing_content(c[:position], c[:url])
```

Comment:
slightly different - that is processing updates from the API, this is updates from crier, subtly different data being processed.